docs/source/getting-started/examples/polars-quantiles.rst

```diff
@@ -0,0 +1,104 @@
+Polars API
+==========
+
+The Polars API for OpenDP is under development and subject to change.
+We are currently focused on supporting a particular use case,
+but we plan to expand the API until it is the preferred interface
+for both Python and R.
+
+The OpenDP Polars API leverages `Polars <https://docs.pola.rs/>`_:
+You'll use Polars methods like ``group_by``, ``agg``, and ``col`` to construct
+most of your query, and only use ``dp`` extensions when needed.
+
+This example demonstrates how to construct DP aggregate statistics, including quantiles.
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. literalinclude:: code/polars-quantiles.rst
+            :language: python
+            :start-after: init
+            :end-before: /init
+
+We need to understand the structure of our private data before we can apply differential privacy,
+but we usually shouldn't or can't look at the private data as we prepare our analysis.
+For this example, our dataset will have three columns:
+
+* ``grouping-key``: integers between 1 and 5; the grouping key
+* ``twice-key``: integers between 2 and 10
+* ``ones``: the float 1.0
+
```

Comment:
More  context for the `twice-key` and `ones` columns may be needed to make the example less abstract; possibly including changing the names

Reply:
I like the thought of using a real-world data set, even if it's just PUMS. This could be the basis of the long-awaited cookbook docs page.